packages/ui/src/memberships/queries/__generated__/members.generated.tsx

```diff
@@ -82,10 +82,10 @@ export type MemberWithDetailsFieldsFragment = {
 }
 
 export type GetMembersQueryVariables = Types.Exact<{
-  where?: Types.Maybe<Types.MembershipWhereInput>
-  orderBy?: Types.Maybe<Array<Types.MembershipOrderByInput> | Types.MembershipOrderByInput>
-  offset?: Types.Maybe<Types.Scalars['Int']>
-  limit?: Types.Maybe<Types.Scalars['Int']>
+  where?: Types.InputMaybe<Types.MembershipWhereInput>
```

Comment:
Also what causes this `Maybe` -> `InputMaybe` in every `queries/__generated__/xxx.generated.tsx`

Reply:
It might be this: [https://github.com/dotansimha/graphql-code-generator/pull/6815#issue-1022447896](https://github.com/dotansimha/graphql-code-generator/pull/6815#issue-1022447896) It looks like something changed int the GraphQL code generator and there's a new type. The definition in generated types is:

```typescript
export type Maybe<T> = T | null;
export type InputMaybe<T> = Maybe<T>;
```